src/components/forms/UpdateResort.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import ResortForm from './ResortForm';
+
+export default class UpdateResortFormPage extends Component {
```

Comment:
How would you handle further merging of `UpdateResortFormPage` and `NewResortFormPage` components?